yaml-tests/src/test/resources/recursive-cte.yamsql

```diff
@@ -68,6 +68,51 @@ test_block:
                  {210, 20},
                  {250, 50},
                  {250, 50}]
+    -
+      - query: with recursive c1 as (
+            select id, parent from t1 where parent = -1
+            union all
+            select b.id, b.parent from c1 as a, t1 as b where a.id = b.parent) select id from c1
+      - supported_version: !current_version
```

Comment:
It looks like without this, some JDBC tests were failing as they were incorrectly trying to run continuations against `4.0.575.0`. If I understand what this configuration means correctly, the query will be tested against current commit onwards only, which is the correct thing to do, considering that recursive CTE support of continuations was misbehaving before this PR. @ScottDugas can you please have a look?